dashboard/src/components/myProgress/MyProjectSkillsPage.vue

```diff
@@ -98,6 +98,18 @@ limitations under the License.
     },
     computed: {
       themeObj() {
+        if (this.$route.query.classicSkillsDisplay) {
```

Comment:
@sudo-may this if statement will evaluate to true for any value (eg,`?classicSkillsDisplay=false`) - the same for `enableTheme`

Reply:
good call! fixed!